lib/openhab/dsl/items/item_delegate.rb

```diff
@@ -28,6 +28,25 @@ def self.extended(base)
           base.include OpenHAB::DSL::Items::ItemDelegate
         end
 
+        #
+        # Delegate the supplied methods to the supplied item
+        #
+        # @param [Object] item
+        # @param [<Type>] *methods <description>
+        #
+        # @return [<Type>] <description>
+        #
+        def def_oh_item_delegator(item, *methods)
```

Comment:
Would this be useful for other Item types as well? It's actually quite inconsistent (at the OH-side) that this method is available on the Item, in most other cases you have to retrieve it's state before doing anything, so not sure this would be of any benefit anywhere else.

Reply:
Not sure.. Are you suggesting I take it out and just override distance_from?

Reply:
I generally like the way you have made these functions generic, but in this case I think it would be enough to override the method. This way it would also be possible to have comparisons with strings, like `LocationItem.distance_from('30, 50')`